config/infra-dev/nextflow-ecs-task-definition.yaml

```diff
@@ -29,3 +29,6 @@ parameters:
 
 stack_tags:
   {{stack_group_config.default_stack_tags}}
+
+sceptre_user_data:
+  environment: !file src/tower/resources/environment.yaml
```

Comment:
This seems ok to me.  If you want to avoid jinja then you can probably pass the file in as a parameter (string type) as well.  For example..

in sceptre file..
```
parameters:
  environment: !file src/tower/resources/environment.yaml.txt
```

then in cfn template..
```
Parameters:
  Environment:
    Type: String

Resources:
  TowerTask:
    Properties:
      ContainerDefinitions:
        - Name: !Sub '${CronContainerName}-MigrateDb'
           Environment: !Ref Environment.yaml
```

Reply:
Will the `!Ref` and `!Sub` resolvers be resolved correctly with this approach? Would that be possible with the combined use of `!Sub` and `!Ref`?